roles/openshift_logging_mux/tasks/main.yaml

```diff
@@ -177,6 +177,48 @@
   check_mode: no
   changed_when: no
 
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
```

Comment:
Could there be any way to obtain the DC name or container name in the ansible task in the common place?  I'd like to replace logging-es-pvc.yml with {{ DCname }}-pvc.yml or logging-{{ containerName }}-pvc.yml here...
    src: pvc.j2
    dest: "{{ tempdir }}/templates/logging-es-pvc.yml"